mmv1/products/privateca/CertificateAuthority.yaml

```diff
@@ -653,6 +653,8 @@ properties:
           subordinate CertificateAuthority. This field is used for information
           and usability purposes only. The resource name is in the format
           `projects/*/locations/*/caPools/*/certificateAuthorities/*`.
+        # certificateAuthority is not included in the response since the sub-CA is activated via specifing pemIssuerChain
+        custom_flatten: 'templates/terraform/custom_flatten/privateca_certificate_authority_subordinate_config_certificate_authority.go.erb'
```

Comment:
Can this be replaced with just ignore_read: true?

Reply:
Started with that. Does not work.

Per https://screenshot.googleplex.com/5zSqfS4ow3wACjo, `ignore_read` is not supported on a nested fields.

Reply:
Ah, yeah